assets/src/application/ui/display/espressoIcons/EspressoIcon.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { svgPath } from './svgPaths';
+import { EspressoIconProps } from './types';
+
+/**
+ * Custom Event Espresso Dashicons
+ */
+const EspressoIcon = React.forwardRef<SVGSVGElement, EspressoIconProps>(
+	({ name, svgSize = 20, className, ...otherProps }, ref) => {
+		const path = svgPath[name];
+
+		if (!path) {
+			return null;
+		}
+
+		const htmlClass = classNames(className, 'dashicon', `dashicons-${name}`, 'espresso-icon');
+
+		return (
+			<svg
+				role={'img'}
+				aria-hidden={true}
+				focusable={false}
+				xmlns='http://www.w3.org/2000/svg'
+				width={svgSize}
+				height={svgSize}
+				viewBox='0 0 20 20'
```

Comment:
this is standard for all icons?
just took a look at GB's, they don't have this set as default: https://github.com/WordPress/gutenberg/blob/master/packages/primitives/src/svg/index.js#L89

Reply:
doesn't this depend on the scale that the SVG path was written in?